src/components/ExitGateCard/ExitGateCard.tsx

```diff
@@ -0,0 +1,114 @@
+import classnames from 'classnames';
+import { ComponentProps, ElementType, forwardRef } from 'react';
+import { getCommonProps } from '../../utils';
+import Button from '../Button/Button';
+import { ButtonVariants } from '../Button/types';
+import { LinkProps } from '../Link';
+import { SeldonImage } from '../SeldonImage';
+import { Text, TextVariants } from '../Text';
+
+export interface ExitGateCardProps extends ComponentProps<'div'> {
+  /**
+   * Image src to display at the right side of the article.
+   */
+  imageSrc?: string;
+  /**
+   * Top label for the article.
+   */
+  label?: string;
+  /**
+   * Header for the article.
+   */
+  header?: string;
+  /**
+   * Description for the article.
+   */
+  description?: string;
```

Comment:
I think these should be `ReactNode` where possible to allow more flexibility, `label`, `header`, and `description` and `linkLabel`